index.html

```diff
@@ -296,6 +292,12 @@
     var daySpan = document.getElementById("day");
     daySpan.firstChild.textContent = dateToString(dateObj);
   }
+  function getFunnyImage(json_obj) {
```

Comment:
json_obj isn't very descriptive. Perhaps call it dayEventObj, or something similar?